### ISSUE TITLE
Advanced tests configuration

### ISSUE THREAD

**Issue by mlewand** [cksource/ckeditor-plugin-a11ychecker#110]
# Overview

We should introduce advanced tests configuration.
## Create Mapping For Test Groups

Developer should be able to easily set WCAG2 A, WCAG AA or WCAG AAA tests without knowing what tests belongs to which group.

e.g.

``` javascript
config.a11ychecker_preset = 'WCAG2AA'
```
## Allow To Exclude/Include Some Tests

It should be possible to add/remove particular tests.

``` javascript
config.a11ychecker_preset = 'WCAG2AA'
config.a11ychecker_include = [ 'foo', 'bar' ];
config.a11ychecker_exclude = [ 'baz' ];
```
